### ISSUE TITLE
Needs to be update to use layout management

### ISSUE THREAD

**Issue by scottmc** [HaikuArchives/Bowser#4]
several clipping and alignment issues throughout the application

**Comment by humdingerb** [comment]
I'd like to point out that Bowser is extremely old. AFAIK it was the ancestor of Vision.
IMO, the effort put into this old thing is misplaced and should rather be spent on Vision, which has come a long way, but is still in need of e.g. proper layout management.

**Comment by janus2** [comment]
I think we should choose wisely on which project spent the scarce resources we have. There are a lot of dups, we should choose the best app and try to improve that. (For example PecoRename, ReName). Sometimes the apps are so old that cost less effort to start from scratch that tries to make usable for the standard expected from the users.

**Comment by scottmc** [comment]
Agreed.  Perhaps we should update the Readmes for the lesser ones, point to the better ones, and just leave them be for reference.
